test/cypress/integration/sidebar-extension.spec.ts

```diff
@@ -71,5 +77,30 @@ context('Sidebar extension', () => {
     })
   })
 
+  it('creates, gets, and deletes a content type through spaceAPI', () => {
```

Comment:
I didn't want to add _all_ the methods for _all_ the entities on space API, because we're not testing the CMA implementation, instead tried to get a full basic example in, so we at least know the integration is working on an SDK level.